engine/access/rpc/backend/backend_transactions.go

```diff
@@ -716,13 +716,7 @@ func (b *backendTransactions) getTransactionResultsByBlockIDFromAnyExeNode(
 	req execproto.GetTransactionsByBlockIDRequest,
 ) (*execproto.GetTransactionResultsResponse, error) {
 	var errs *multierror.Error
-	logAnyError := func() {
```

Comment:
I think we should keep this. It's intended to make sure we log errors from failing nodes even if the call succeeded

Reply:
I can keep it but it only logs the errors that come from the `multierror`. Or should it log the not found error and the error when it's passed 0 execution nodes as well?

Reply:
it should just log the multierror. we just want to make sure we log errors from ENs even if we eventually succeed